viral/learning_stages.py

```
@@ -22,112 +29,139 @@
+    )
+
+    spks = np.load(
+        TIFF_UMBRELLA
+        / session.date
+        / session.mouse_name
+        / "suite2p"
+        / "plane0"
+        / "oasis_spikes.npy"
+    )
+
+    pcs_mask, _ = get_place_cells(
```

Ruff (0.14.0)

Unpacked variable `pcs_mask` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)